gateway/gateway.go

```diff
@@ -1248,54 +1249,63 @@ func (gateway *HandleT) StartWebHandler(ctx context.Context) error {
 	gateway.backendConfig.WaitForConfig(ctx)
 	gateway.logger.Infof("WebHandler Starting on %d", webPort)
 	component := "gateway"
-	srvMux := mux.NewRouter()
+	srvMux := chi.NewRouter()
 	srvMux.Use(
-		gorillaware.StatMiddleware(ctx, srvMux, stats.Default, component),
+		chiware.StatMiddleware(ctx, srvMux, stats.Default, component),
 		middleware.LimitConcurrentRequests(maxConcurrentRequests),
 		middleware.UncompressMiddleware,
 	)
-	internalMux := srvMux.PathPrefix("/internal").Subrouter() // mux for internal endpoints
-
-	srvMux.HandleFunc("/v1/batch", gateway.webBatchHandler).Methods("POST")
-	srvMux.HandleFunc("/v1/identify", gateway.webIdentifyHandler).Methods("POST")
-	srvMux.HandleFunc("/v1/track", gateway.webTrackHandler).Methods("POST")
-	srvMux.HandleFunc("/v1/page", gateway.webPageHandler).Methods("POST")
-	srvMux.HandleFunc("/v1/screen", gateway.webScreenHandler).Methods("POST")
-	srvMux.HandleFunc("/v1/alias", gateway.webAliasHandler).Methods("POST")
-	srvMux.HandleFunc("/v1/merge", gateway.webMergeHandler).Methods("POST")
-	srvMux.HandleFunc("/v1/group", gateway.webGroupHandler).Methods("POST")
-	srvMux.HandleFunc("/health", WithContentType("application/json; charset=utf-8", app.LivenessHandler(gateway.jobsDB))).Methods("GET")
-	srvMux.HandleFunc("/", WithContentType("application/json; charset=utf-8", app.LivenessHandler(gateway.jobsDB))).Methods("GET")
-	srvMux.HandleFunc("/v1/import", gateway.webImportHandler).Methods("POST")
-	srvMux.HandleFunc("/v1/audiencelist", gateway.webAudienceListHandler).Methods("POST")
-	srvMux.HandleFunc("/pixel/v1/track", gateway.pixelTrackHandler).Methods("GET")
-	srvMux.HandleFunc("/pixel/v1/page", gateway.pixelPageHandler).Methods("GET")
-	srvMux.HandleFunc("/v1/webhook", gateway.webhookHandler.RequestHandler).Methods("POST", "GET")
-	srvMux.HandleFunc("/beacon/v1/batch", gateway.beaconBatchHandler).Methods("POST")
-	srvMux.PathPrefix("/v1/warehouse").Handler(http.HandlerFunc(warehouseHandler)).Methods("GET", "POST")
-	srvMux.HandleFunc("/version", WithContentType("application/json; charset=utf-8", gateway.versionHandler)).Methods("GET")
-	srvMux.HandleFunc("/robots.txt", gateway.robots).Methods("GET")
-
+	internalMux := srvMux.Route("/internal", func(r chi.Router) {
+		// mux for internal endpoints
+	})
```

Comment:
```suggestion
	srvMux.Route("/internal", func(r chi.Router) {
		r.Post("/v1/extract", gateway.webExtractHandler)
	})
```
Should we follow this pattern for consistency?